src/data/h1/index.js

```diff
@@ -4,18 +4,18 @@ const path = require("path");
 const glob = require("glob");
 const {getDirectReferencedTagNames} = require("./invader");
 
-const basicTagsList = glob.sync(path.join(__dirname, "tags", "*.yml")).map(tagFileName => {
-  return yaml.safeLoad(fs.readFileSync(tagFileName, "utf8"));
-});
-
-const {workflowItems, similarItems, workflows} = yaml.safeLoad(fs.readFileSync(path.join(__dirname, "tools.yml"), "utf8"));
-
 function strAsList(strOrList) {
   if (!strOrList) return null;
   return typeof(strOrList) === "string" ? [strOrList] : strOrList;
 }
 
 function buildData(invaderStructDefs) {
+  const basicTagsList = glob.sync(path.join(__dirname, "tags", "*.yml")).map(tagFileName => {
+    return yaml.safeLoad(fs.readFileSync(tagFileName, "utf8"));
+  });
+
+  const {workflowItems, similarItems, workflows} = yaml.safeLoad(fs.readFileSync(path.join(__dirname, "tools.yml"), "utf8"));
```

Comment:
Moving these here means the YML files are loaded every site build rather than every process startup, which is needed for automatic rebuilds during dev mode.